target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -155,9 +156,6 @@ ramips_setup_interfaces()
 	tplink,tl-wpa8631p-v3)
 		ucidef_set_interface_lan "lan1 lan2 lan3 plc0"
 		;;
-	tplink,mr600-v2-eu)
-		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3" "wan"
-		;;
```

Comment:
The default version also configures the `lan4` interface in addition. 
```
ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4" "wan"
```
I think you have to keep this special case.

Reply:
The case of tplink,mr600-v2-eu was not removed, but moved to another place in the code.